x/ccv/provider/keeper/grpc_query_test.go

```diff
@@ -322,22 +323,31 @@ func TestGetConsumerChain(t *testing.T) {
 		{},
 	}
 
+	metadataLists := make([]types.ConsumerMetadata, len(chainIDs))
+	consumerIDs := make([]string, len(chainIDs))
+
+	// skip first consumerID
+	pk.FetchAndIncrementConsumerId(ctx)
```

Comment:
Why are we skipping the first consumer id?

Reply:
Removed.